data/human/news.txt

```diff
@@ -236,6 +237,31 @@ news "street vendor"
 			"I wish I had enough money to ask captains roaming around the spaceport for transport to somewhere nicer."
 			"Sell a man a fish and you can sell him fish every day; teach a man to fish and you lose your customer."
 			"I have the safest stall in town; nobody ever died from the food that I serve. At least, nobody that I know."
+			"Can I offer you an egg in this trying time?"
+		word
+			`"`
+
+
+
+news "grifter"
+	location
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+		attributes "urban"
+		not attributes "paradise"
+	name
+		word
+			"Suspicious salesperson"
+			"Hooded salesperson"
+			"Shifty dealer"
+	message
+		word
+			`"`
+		word
+			"Can I interest you in a pair of super shoes? You'll lap the competition with these sneakers! They can also help when walking away from people who are promoting shady products!"
```

Comment:
The last phrase in this line seems a touch odd? I would suggest rephrasing or removing it.

Reply:
I actually like that last line; the person speaking it does seem to have some self-reflection. ;)